### PR TITLE
chore: deprecate learner view

### PR DESCRIPTION
**Pull request by alangsto**
Remove all code associated with the learner view.

### CODE REVIEW

File: analytics_dashboard/courses/views/__init__.py [hunk 1]
```diff
@@ -745,22 +693,6 @@ def get_context_data(self, **kwargs):
 
         context['page_data'] = self.get_page_data(context)
 
-        # Some orgs do not wish to allow access to learner analytics.
-        # See https://openedx.atlassian.net/browse/DENG-536
-        course_org = CourseKey.from_string(self.course_id).org
-        if course_org in settings.BLOCK_LEARNER_ANALYTICS_ORG_LIST:
```
Comment (ashultz0) [inline analytics_dashboard/courses/views/__init__.py:751]: we should scrub out this setting later, it exists in edx-internal

**Review by ashultz0 — APPROVED** [review]
one note for further deprecation cleanup we can do later, this looks good

File: analytics_dashboard/courses/views/__init__.py (hunk 1 above)
Comment (alangsto) [reply analytics_dashboard/courses/views/__init__.py:751]: sounds good, I will leave in code that relates to this setting then

Comment (alangsto) [reply analytics_dashboard/courses/views/__init__.py:751]: Created https://github.com/edx/edx-internal/pull/6880 for removing this settings from edx-internal

**Pull request merged**